doc/faq/virtualenv_faq.rst

```diff
@@ -40,6 +43,7 @@ exist. Some of these are given here:
 * The ``TKAgg`` backend doesn't require any external dependencies and is
   normally always available.
 * The ``QT4`` framework ``PySide`` is pip installable.
+* ``PYQT5`` is pip installable on Python 3.5.
 * The upcoming `WX Phoenix <http://wiki.wxpython.org/ProjectPhoenix>`_ toolkit
   is ``pip`` installable.
```

Comment:
I know it is not part of your changeset, but this line about WX is highly misleading, as the Pypi link shows:
https://pypi.python.org/pypi/wx/3.0.3
It is unofficial, from a snapshot, and it requires lots of arcane dependencies.  I don't think we should call it "pip installable" without at least some qualification.